Libraries/LibWeb/WebAssembly/Error.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2024, Pavel Shliak <shlyakpavel@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibJS/Runtime/Error.h>
+
+namespace Web::WebAssembly {
+
+class CompileError final : public JS::Error {
```

Comment:
This is not quite right. We need to have prototypes and constructors for these that inherit from ErrorPrototype and ErrorConstructor as well.